src/data_management/rasters/PaginationBar.js

```diff
@@ -2,36 +2,174 @@ import React, { Component } from "react";
 import styles from "./PaginationBar.css";
 
 class PaginationBar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { navigatorInError: false };
+  }
+
+  // render all the page numbers of the list of links
+  // par=loadRastersOnPage is optional
+  renderLinks(links, page, loadRastersOnPage) {
+    return links.map(link => {
+      if (page === link) {
```

Comment:
Currently, the "..." is also blue and has cursor: pointer;
I would suggest to make the "..." also black without cursor: pointer, like the current page number.
```
      if (page === link) {
        return <div key={link}>{link}</div>;
      } else if (link === "...") {
        return <div key={link}>{link}</div>;
      } else {
```

Reply:
Yep ook terecht